### PR TITLE
Release configuration and mvnw 

### PR DESCRIPTION
**Pull request by pavolloffay**
This should configure the same release process as in `opentracing-java` repo.

https://github.com/opentracing-contrib/java-jaxrs/issues/4

### CODE REVIEW

**Comment by pavolloffay** [comment]
I have a question about release rights. Should I register somewhere or can I just copy over values from https://github.com/opentracing/opentracing-java/blob/master/.travis.yml#L42 and it should work? 

@bensigelman @adriancole

**Comment by adriancole** [comment]
you will need to copy the travis directory and .travis file, settings.xml (and corresponding release entries if not already in pom.xml). Also, copy the RELEASING.md

then, you need to get access sorted and re-encrypt values in those files.

**Comment by adriancole** [comment]
request for your sonatype user access here
https://issues.sonatype.org/browse/OSSRH-19982

join bintray then ben or yuri can add you here
https://bintray.com/opentracing/organization/edit

**Comment by pavolloffay** [comment]
@wu-sheng @yurishkuro could you please review? I guess it would be good to exclude integration tests and from release.

**Comment by wu-sheng** [comment]
@pavolloffay , I am not familiar with the release steps. Sorry. Ask @bensigelman or @yurishkuro , I think they know.

**Comment by pavolloffay** [comment]
PR updated test and example artifacts are excluded from publishing to maven: 

```
      <plugin>
        <artifactId>maven-deploy-plugin</artifactId>
        <version>${version.maven-deploy-plugin}</version>
        <configuration>
          <skip>true</skip>
        </configuration>
      </plugin>
```

hopefully it will work.

**Comment by pavolloffay** [comment]
@sjoerdtalsma @bensigelman could you please look at this? 

Do I have to add OT license header in this repo?

File: pom.xml [hunk 1]
```diff
@@ -48,4 +81,77 @@
...
   </dependencyManagement>
+
+  <profiles>
+    <profile>
+      <id>release</id>
+
+      <distributionManagement>
+        <repository>
+          <id>bintray</id>
+          <url>https://api.bintray.com/maven/opentracing/maven/opentracing-java/;publish=1</url>
```
Comment (sjoerdtalsma) [inline pom.xml]: Shouldn't this be `https://api.bintray.com/maven/opentracing/maven/java-jaxrs/;publish=1` ?

**Note:** I'm definately no expert on this; my release actually isn't working yet either: https://github.com/opentracing-contrib/java-globaltracer/issues/2

Comment (pavolloffay) [inline pom.xml]: thanks :+1: !! Yes it should be the package name. https://bintray.com/docs/api/#_maven_upload

**Comment by bensigelman** [comment]
@pavolloffay 

> Do I have to add OT license header in this repo?

I would like the repo to be ASF2.0... I was always under the impression that license headers were not strictly necessary, but I am by no means an expert in these matters and there are certainly plenty of JVM projects that would suggest otherwise :)

**Comment by bensigelman** [comment]
LGTM... if you are able to test manually (e.g., via the steps Adrian added to the OT-java repo), that would seem prudent. If not, we can merge and find out what happens. :)

**Pull request merged**